extensions/amp-story/1.0/amp-story.js

```diff
@@ -349,6 +348,7 @@ export class AmpStory extends AMP.BaseElement {
         `amp-story-page#${escapeCssSelectorIdent(pageId)}`
       );
       page.setAttribute('active', '');
+      page.setAttribute('i-amphtml-initial', '');
```

Comment:
This attribute is very broad and could mean literally anything, please be a bit more specific and/or consistent with how things are named in the JavaScript to make it easier